build.sbt

```
@@ -38,7 +38,7 @@ lazy val core = project
     )
   )
 
-val muV = "0.28.0+42-bd246ae6-SNAPSHOT" // TODO update when mu-scala is released
+val muV = "0.29.1" // TODO update when mu-scala is released
```

I guess we can remove the comment
```suggestion
val muV = "0.29.1"
```